test/uritemplate-test.js

```diff
@@ -3,7 +3,8 @@ var template, expect, examples;
 if (typeof require !== 'undefined') {
   template = require('../lib/url-template.js');
   expect = require("expect.js");
-  examples = require('../uritemplate-test/spec-examples-by-section.json');
+  examples = {}
+  //examples = require('../uritemplate-test/spec-examples-by-section.json');
```

Comment:
commenting this line out allows the local specs to run

Reply:
Why does this need to be commented out for the tests to run?

Reply:
I'm not sure if it's because this is a fork, but that directory was empty. So it was pointing to a test dir with no specs for some reason.

Reply:
Mocha got updated as a part of this, so maybe it handles it differently now. I would delete the line instead of commenting it out. It's in git history if we want it.

Reply:
@brrusselburg looks like a submodule got deleted that has these tests. Can you look into that?